api-docs/api-reference/methods/token/post-authenticate-as-user-with-password-or-api-key-v2.0-tokens.rst

```diff
@@ -190,10 +190,10 @@ Example: Authenticate as user with password XML request
 .. code::
 
    {"auth":
-   	{"passwordCredentials":
-   		{"username":"demoAuthor",
-   		  "password":"myPassword01"}
-   	}
+     {"passwordCredentials":
```

Comment:
I think this would look better formatted this way.
```
{
    "passwordCredentials": {
        "username":"demoAuthor",
        "password":"myPassword01"
    }
}
```